app/helpers/hyku_knapsack/application_helper.rb

```diff
@@ -2,5 +2,12 @@
 
 module HykuKnapsack
   module ApplicationHelper
+    def tenant_registered_curation_concern_types
```

Comment:
this seems unneeded, can't we just check or uncheck work types in the UI? that's the point of that generic feature - to not need a customer specific one.

Reply:
I don't like it either, but managing it this way means that other tenants can't accidentally use it. By default, when a new tenant is created, aren't all of the work types automatically checked? 

Ideally I would have liked to make an option to define the default work types in the tenant's settings, but budget didn't allow going to that extent at this point.

Reply:
Ultimately, I agree with Rob here. 

I really like your idea LaRita of being able to configure a default per tenant, but the timeline being what it is, I'd rather train the client than hard-code it in this way.

Reply:
The other reason for some of the differentiation is to allow for finding the appropriate "default work type" for Bulkrax imports. That varies by tenant, even though some of the tenant specific logic is possibly excessive.